### ISSUE TITLE
month name not in user's locale 

### ISSUE THREAD

**Issue by britiscurious** [RhetTbull/osxphotos#113]
When executing:

./osxphotos export --directory '{created.year}/{created.mm}-{created.month}' DEST

the corresponding subfolders for the created month are created in English although it should be German in my case.

running locale in Terminal (zsh) returns:

LANG="de_DE.UTF-8"
LC_COLLATE="de_DE.UTF-8"
LC_CTYPE="de_DE.UTF-8"
LC_MESSAGES="de_DE.UTF-8"
LC_MONETARY="de_DE.UTF-8"
LC_NUMERIC="de_DE.UTF-8"
LC_TIME="de_DE.UTF-8"
LC_ALL=

**Comment by RhetTbull** [comment]
Thanks for the bug report!  I’m surprised by this because I’m using datetime’s strftime() whose [documentation](https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior) states it uses user’s locale. I’ll dig into the code and see what I can figure out! My intent was that those template substitutions all happen in the user’s locale.

**Comment by britiscurious** [comment]
OK, I did a quick experiment in the shell with python 3 (3.7.3).

```
import locale
locale.getlocale()
```
returns ('de_DE', 'UTF-8')

`locale.getlocale(locale.LC_TIME)`
returns (None, None)

```
import time
from datetime import date
today = date.today()
today.strftime("%A")
```

returns 'Sunday' <- not what I want

`locale.setlocale(locale.LC_TIME, 'de_DE')`
returns 'de_DE'

`locale.getlocale()`
returns ('de_DE', 'UTF-8')


`locale.getlocale(locale.LC_TIME)`
**now** returns (de_DE', 'ISO8859-1')

`today.strftime("%A")`
**now** returns 'Sonntag'

**Comment by RhetTbull** [comment]
@britiscurious thanks -- that's helpful and I think I'm on the right path now.  Try your example above but execute this line first:

`locale.setlocale(locale.LC_ALL,"")`

I did some similar testing after seeing your examples, changing my locale to de_DE and got same results.  Not sure why, but I think the line above will get datetime to use the right locale as in example below:

```sh
[I] ➜ locale
LANG="de_DE.UTF-8"
LC_COLLATE="de_DE.UTF-8"
LC_CTYPE="de_DE.UTF-8"
LC_MESSAGES="de_DE.UTF-8"
LC_MONETARY="de_DE.UTF-8"
LC_NUMERIC="de_DE.UTF-8"
LC_TIME="de_DE.UTF-8"
LC_ALL=
```


```python
[I] ➜ python3
>>> import locale
>>> import datetime
>>> now = datetime.datetime.now()
>>> now.strftime("%A")
'Sunday'
>>> locale.setlocale(locale.LC_ALL,"")
'de_DE.UTF-8'
>>> now.strftime("%A")
'Sonntag'
```

```python
[I] ➜ python3 -i examples/photos_repl.py
osxphotos version: 0.28.5
Using last opened Photos library: /Users/rhet/Pictures/Photos Library.photoslibrary
loading database
done: took 2.48562013 seconds
database version: 6000
getting photos
found 25964 photos in 2.3432144280000005 seconds
>>> from osxphotos.template import render_filepath_template
>>> import locale
>>> photos[0].date
datetime.datetime(2017, 1, 31, 16, 19, 19, 192088, tzinfo=datetime.timezone(datetime.timedelta(days=-1, seconds=57600)))
>>> render_filepath_template("{created.month}",photos[0])
(['January'], [])
>>> locale.setlocale(locale.LC_ALL,"")
'de_DE.UTF-8'
>>> render_filepath_template("{created.month}",photos[0])
(['Januar'], [])
>>>
```

**Comment by britiscurious** [comment]
Just wanted to confirm that
`locale.setlocale(locale.LC_ALL,"")`
does the trick here, too.

**Comment by RhetTbull** [comment]
Excellent!  I think I've fixed this -- working on test cases now, will push update to github later today.

```sh
[I] ➜ locale
LANG="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_CTYPE="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_ALL=
[I] ➜ python3 -m osxphotos export ~/Desktop/export --directory '{created.year}/{created.mm}-{created.month}' --uuid 9D0CE36F-831A-4F5D-83E0-A5473B45EB64 --verbose
Using last opened Photos library: /Users/rhet/Pictures/Photos Library.photoslibrary
Exporting 1 photo to /Users/rhet/Desktop/export...
Exporting 9D0CE36F-831A-4F5D-83E0-A5473B45EB64.jpeg as IMG_1321.JPG
Exported 9D0CE36F-831A-4F5D-83E0-A5473B45EB64.jpeg to ['/Users/rhet/Desktop/export/2020/03-March/IMG_1321.JPG']
```

```sh
[I] ➜ locale
LANG="de_DE.UTF-8"
LC_COLLATE="de_DE.UTF-8"
LC_CTYPE="de_DE.UTF-8"
LC_MESSAGES="de_DE.UTF-8"
LC_MONETARY="de_DE.UTF-8"
LC_NUMERIC="de_DE.UTF-8"
LC_TIME="de_DE.UTF-8"
LC_ALL=
[I] ➜ python3 -m osxphotos export ~/Desktop/export --directory '{created.year}/{created.mm}-{created.month}' --uuid 9D0CE36F-831A-4F5D-83E0-A5473B45EB64 --verbose
Using last opened Photos library: /Users/rhet/Pictures/Photos Library.photoslibrary
Exporting 1 photo to /Users/rhet/Desktop/export...
Exporting 9D0CE36F-831A-4F5D-83E0-A5473B45EB64.jpeg as IMG_1321.JPG
Exported 9D0CE36F-831A-4F5D-83E0-A5473B45EB64.jpeg to ['/Users/rhet/Desktop/export/2020/03-März/IMG_1321.JPG']
```

**Comment by RhetTbull** [comment]
Fixed in version 0.28.6
